src/extension/src/client-modules/generation.ts

```diff
@@ -1,25 +1,17 @@
 import * as vscode from "vscode";
 import { WizardServant, IPayloadResponse } from "../wizardServant";
+import { CONSTANTS } from "../constants/constants";
+import { TelemetryEventName } from "../constants/telemetry";
+import { ITelemetryService } from "../telemetry/telemetryService";
 import {
-  CONSTANTS,
-  AzureResourceType
-} from "../constants/constants";
-import { TelemetryEventName } from '../constants/telemetry';
-import { IActionContext, ITelemetryService } from "../telemetry/telemetryService";
-import { AzureServices } from "../azure/azureServices";
-import { CoreTemplateStudio } from "../coreTemplateStudio";
-import { ResourceGroupSelection } from "../azure/azure-resource-group/resourceGroupModule";
-import { Logger } from "../utils/logger";
-import { IGenerationPayloadType, IServicesGenerationPayload } from "../types/generationPayloadType";
-import { sendToClientGenerationStatus, GenerationItemStatus, updateStatusMessage, GENERATION_NAMES } from "../utils/generationStatus";
-import { MESSAGES } from "../constants/messages";
+  IAppService,
+  ICosmosDB,
+  IGenerationData,
+  SERVICE_CATEGORY,
+  SERVICE_TYPE,
+} from "../types/generationPayloadType";
```

Comment:
Is generationPayloadType still a valid file name or maybe just generationType could work?